src/operators.jl

```diff
@@ -308,12 +308,13 @@ const _bindings = Dict() # XXX GC Issue? can't use WeakKeyDict with Pairs...
 const _active_binds = Dict()
 
 """
-    `bind!(dest, src, twoway=true)`
+    `bind!(dest, src, twoway=true, initial=true)`
```

Comment:
`bind!(dest, src, twoway=true; initial=true)` I guess it should have the semi-colon?